docs/pages/admin-guides/access-controls/login-rules/guide.mdx

```diff
@@ -6,8 +6,23 @@ labels:
  - zero-trust
 ---
 
-This guide will walk you through the process of writing, testing, and adding the
-first Login Rule to your Teleport cluster.
+Login Rules define logic that transforms the external traits of a user who signs
+into Teleport, providing consistent access control policies for users in your
```

Comment:
A key use case for login rules is when you the team who manages Teleport is not the same as the team who manages the IdP. In this case, it's often easier to use login rules to make transformations to the traits prior to Teleport's SSO process than to request changes in the IdP configuration.